_data/wildfly-categories.yaml

```diff
@@ -135,3 +135,6 @@ categories:
   - name: WildFly Galleon
     id: wf-galleon
     description: Provision WildFly with Galleon Feature Packs and Layers
+  - name: WildFly AI
```

Comment:
This shouldn't block, but I think simply 'AI' is better. These are meant to be general categories. WildFly Core is kind of an exception because 'Core' is too vague. 'WildFly Elytron' is the name of a large scale project, and anyway most anything using it should use 'security' as well. 'WildFly Galleon' is probably a mistake and could just be 'Galleon' or just 'Provisioning'.